src/web/routes/application/are-you-pregnant/validate.js

```diff
@@ -43,12 +35,14 @@ const validateExpectedDeliveryDate = (expectedDeliveryDate, { req }) => {
 
 const validate = [
   check('areYouPregnant').isIn([YES, NO]).withMessage(translateValidationMessage('validation:selectYesOrNo')),
-  addExpectedDeliveryDateToBody,
+  /*
+    First parameter here is used for setting the error in the view only. The value to validate is constructed from
+    multiple fields in the validation method itself.
+  */
   check('expectedDeliveryDate').custom(validateExpectedDeliveryDate)
```

Comment:
I find this more confusing with the comment. I think its fairly self explanatory without the comment.

Reply:
Ok, I'll remove